doc/onboarding.md

```diff
@@ -15,8 +15,7 @@ onboarding session.
 ## Fifteen minutes before the onboarding session
 
 * Prior to the onboarding session, add the new Collaborator to
-  [the Collaborators team](https://github.com/orgs/nodejs/teams/collaborators),
-  and to [the Members team](https://github.com/orgs/nodejs/teams/members) if
+  [the Collaborators team](https://github.com/orgs/nodejs/teams/collaborators) if
```

Comment:
Sorry I didn't think of this before but they will *not* already be a part of the nodejs/collaborators team, so the `if they are not already part of it` can be removed too. :-D

Reply:
(The `if they are not already part of it` applied to nodejs/members.)

Reply:
No worries, I've made the change now.